hypertrace-graphql-labels-schema-impl/src/main/java/org/hypertrace/graphql/label/dao/LabelResponseConverter.java

```diff
@@ -60,5 +66,36 @@ private Label convertLabel(org.hypertrace.label.config.service.v1.Label label) {
     String key;
     String color;
     String description;
+    Map<String, List<LabeledEntity>> labeledEntitiesMap;
+
+    @Override
+    public LabeledEntityResultSet labeledEntities(String entityType, int limit) {
+      List<LabeledEntity> labeledEntities =
+          Optional.ofNullable(labeledEntitiesMap.get(entityType)).orElse(Collections.emptyList());
+      List<LabeledEntity> labeledEntitiesWithLimit =
+          labeledEntities.size() > limit ? labeledEntities.subList(0, limit) : labeledEntities;
+      return new DefaultLabeledEntityResultSet(
+          labeledEntitiesWithLimit, labeledEntitiesWithLimit.size(), labeledEntities.size());
+    }
+  }
+
+  @Value
+  @Accessors(fluent = true)
+  private static class DefaultLabeledEntityResultSet implements LabeledEntityResultSet {
+    List<LabeledEntity> results;
+    long count;
+    long total;
+  }
+
+  @Value
+  @Accessors(fluent = true)
+  private static class DefaultLabeledEntity implements LabeledEntity {
+    String id;
+    Map<String, Object> attributeValues;
```

Comment:
ah crap, I have an enormous change (oops) locally that changes how attributes work

Reply:
let me know what changes are coming, and I will update accordingly

Reply:
The relevant part here is that `AttributeQueryable` changes.
```
Object attribute(@GraphQLName(AttributeExpression.ARGUMENT_NAME) AttributeExpression expression);
```
and most of the impls are now holding a 
```
    Map<AttributeExpression, Object> attributeValues;
```

Unless I'm able to get stuff merged (and it won't be today), not much you can do to prepare for it, since don't have the new types.